internal/services/connections/api_connection_resource.go

```diff
@@ -90,6 +91,17 @@ func resourceConnection() *pluginsdk.Resource {
 			"tags": commonschema.Tags(),
 		},
 	}
+
+	if !features.FourPointOhBeta() {
+		resource.Schema["display_name"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+		}
+	}
```

Comment:
how come we are removing the default and making it computed in 4.0?

Reply:
Seems you missed the something. This PR is removing Computed and add default value in 4.0.